vue3-apollo-quasar/src/views/Repository/RepositoryDetails.vue

```diff
@@ -10,29 +9,37 @@
       :forks="forkCount"
       :issuesCount="openIssueCount"
       :pullRequestsCount="openPullRequestCount"
-    />
-    <section class="q-mx-auto q-my-xl code-section">
-      <q-card flat bordered>
-        <FileExplorer v-if="!fileTree.isBlob" :content-list="fileTree.data" />
-        <pre class="file-text" v-else>{{ fileTree.text }}</pre>
-      </q-card>
-    </section>
+    >
+      <template #code>
+        <Code
+          :owner="owner"
+          :repo="repo"
+          :dirpath="dirpath"
```

Comment:
below you have something similar `repo-dir-path` but above it's one word, `dirpath`. Consistency is key. 

Could we pick one format? I'd suggest `dirPath` which would resolve to `dir-path` when accessing the prop.